Automattic/wp-calypso
# Move "Other" category to bottom in email stats for devices & clients

#### Proposed Changes

This PR moves the _Other_ category to the bottom of the devices & clients lists.

#### Testing Instructions

* Visit `/stats/email/open/<site>/day/<postid>?flags=newsletter/stats`. You can add the startDate parameter, to load the stats for a specific date.
* Ensure the "Other" category is moved to the bottom for devices & clients:

| Before | After |
|-------|-------|
| <img width="534" alt="CleanShot 2023-01-04 at 15 50 44@2x" src="https://user-images.githubusercontent.com/528287/210581561-b325cf2c-14b6-496d-9b2e-8c28c4442082.png"> |<img width="535" alt="CleanShot 2023-01-04 at 15 44 32@2x" src="https://user-images.githubusercontent.com/528287/210580545-d2afa12a-ebef-4edd-821b-a89e59a258b7.png"> |

#### Pre-merge Checklist
